src/gui/transferlistwidget.cpp

```diff
@@ -807,6 +812,38 @@ void TransferListWidget::askAddTagsForSelection()
         addSelectionTag(tag);
 }
 
+void TransferListWidget::editTorrentTrackers()
+{
+    const QVector<BitTorrent::TorrentHandle *> torrents = getSelectedTorrents();
+    QVector<BitTorrent::TrackerEntry> commonTrackers;
+
+    if (!torrents.empty()) {
+        commonTrackers = torrents[0]->trackers();
+
+        for (const BitTorrent::TorrentHandle *torrent : torrents) {
+            QSet<BitTorrent::TrackerEntry> trackerSet;
+
+            for (const BitTorrent::TrackerEntry &entry : asConst(torrent->trackers()))
+                trackerSet.insert(entry);
+
+            commonTrackers.erase(std::remove_if(commonTrackers.begin(), commonTrackers.end()
```

Comment:
I would use QSet for commonTrackers and just apply `commonTrackers.intersect(trackerSet);` here.

Reply:
That would not preserve the original order, isn't?

Reply:
I don't think this order is so important since it might be different for different torrents. E.g.:
Torrent1
- Tracker1
- Tracker2
- Tracker3
Torrent2
- Tracker3
- Tracker1
- Tracker2
- Tracker4
Torrent3
- Tracker4
- Tracker1
- Tracker3

Common trackers are Tracker1 and Tracker3 but the order is different.

Reply:
but what if the user already managed to have the trackers in the same order and wish to maintains it?

Reply:
Yeah, that makes sense. All right, let it stay as it is.